core/src/test/java/io/confluent/kafka/schemaregistry/RestApp.java

```diff
@@ -48,8 +49,9 @@ public RestApp(int port, String zkConnect, String kafkaTopic,
                  String compatibilityType, boolean masterEligibility) {
     prop = new Properties();
     prop.setProperty(SchemaRegistryConfig.PORT_CONFIG, ((Integer) port).toString());
-    prop.setProperty(SchemaRegistryConfig.KAFKASTORE_CONNECTION_URL_CONFIG, zkConnect);
-    prop.put(SchemaRegistryConfig.KAFKASTORE_TOPIC_CONFIG, kafkaTopic);
+    prop.setProperty(KafkaStoreConfig.KAFKASTORE_CONFIG_PREFIX + KafkaStoreConfig.KAFKASTORE_CONNECTION_URL_CONFIG, zkConnect);
+    prop.put(KafkaStoreConfig.KAFKASTORE_CONFIG_PREFIX + KafkaStoreConfig.KAFKASTORE_TOPIC_CONFIG, kafkaTopic);
+    prop.setProperty(KafkaStoreConfig.KAFKASTORE_CONFIG_PREFIX + KafkaStoreConfig.KAFKASTORE_KAFKA_GROUPID, "test-group");
```

Comment:
This group ID seems to be replacing some setup we previously did in Kafka store, but this doesn't match the values it previously did for the schema registry service. This would be a compatibility issue since that ID is used to uniquely identify the instance. See comment later on for more details.